pmm/pmm2-ovf-test.groovy

```diff
@@ -58,7 +59,7 @@ pipeline {
             description: 'Postgre SQL Server version',
             name: 'PGSQL_VERSION')
         string(
-            defaultValue: '--addclient=ps,1 --addclient=mo,1',
+            defaultValue: '--addclient=haproxy,1 --addclient=ps,1',
```

Comment:
not using haproxy too, lets remove this from setup

Reply:
done, thanks